index.html

```diff
@@ -671,10 +671,241 @@
+					it is exposed by allowing users to scroll to it.
+					This behavior is typical of most desktop or mobile web browsers.</p>
+					<p>In paginating media, content is broken up into discrete pages,
+					and content that overflows one page is displayed on subsequent pages.
+					This behavior is typical of web content printed on paper,
+					of the output of web-to-PDF renderers,
+					and of many e-book reading systems.</p>
+					<p>Some User Agents may support both scrolling and pagination.
+					As the preferences of individual readers vary,
+					and as different types of publications may be better suited for one or the other,
+					this specification encourages User Agents to support both,
+					and to offer a choice to their users.</p>
+
+					<div class="note">
+					<p>Authors or Publishers occasionally have strong preferences between scrolling and paginating for particular types of content,
+					and may wish to require a publication to always be rendered in one or the other mode.
+					This wish is not, however, something that <a>Web Publication</a> User Agents can effectively enforce.
+					Since <a>Web Publications</a> are composed of backwards-compatible ordinary web content,
+					legacy User Agents unaware of this specification will be able to open them,
+					and will render them as they render any other content.
+					Even <a>Web Publication</a> aware User Agents may only support for one of the two layout modes,
+					and cannot be expected to honor requests to use the other one.
+
+					<p>However, within the limits of the layout features supported by the user Agent used to render a publication,
+					it is possible for authors to provide a page-like experience on scrolling media,
+					using css features such as those defined in [[CSS-MULTICOL-1]] or [[CSS-SCROLL-SNAP-1]],
+					or the <a href="https://www.w3.org/TR/css-values-3/#viewport-relative-lengths">viewport units</a> defined in [[CSS-VALUES-3]].
+					</div>
+
+					<div class=issue>
+					<p>It may nonetheless be useful for authors to be able to specify a preference between scrolling and pagination,
+					even if a strict requirement is not possible. This should most likely be addressed through an extension of
+					<a href="https://www.w3.org/TR/css-device-adapt-1/#atviewport-rule"><code>@viewport</code></a>
+					or of <a href="https://drafts.csswg.org/css-device-adapt/#viewport-meta">the viewport meta tag</a>(see [[CSS-DEVICE-ADAPT]]),
+					or possibly through an extension of <a href="https://drafts.csswg.org/css-page-3/#at-page-rule"><code>@page</code></a> (See [[CSS-PAGE-3]]).
+					This should be discussed with the relevant working groups (<a href="https://www.w3.org/Style/CSS/">CSSWG</a>, <a href="https://www.w3.org/WebPlatform/WG/">WebPlatformWG</a>, <a href="https://whatwg.org/">WHATWG</a>).
+					</div>
+				</section>
+				<section id="enh-pagination">
+					<h4>Paginated Layout</h4>
+
```

Comment:
I think we should keep this out for now...it's just too controversial

Reply:
I think it's useful to track these issues here at this stage. The web will need to come to terms with pagination, and with "collections" of documents. When our spec is more mature it may make sense to move some of these to CSS, but I think it's too soon for that.

Reply:
Would you mind elaborating?

I am not claiming to have the perfect answer, but I think this is a topic we cannot avoid. My goal in writing this section is to say something concrete, even if minimal, so that we can begin working out what it is we agree or disagree about.

As far as I can tell, there are three things we have to have:
* How do you describe a collection of documents/resources
* How do you render the thing
* Where/how do you store the metadata about the collection

If we don't have that, we do not have a specification (although I suppose you could skip the metadata part and still have a specification, just not one we'd be very happy with).

Without a collection and working with a single document/resource, we already know how that works, it's just HTML and CSS. Once you have a collection, we no longer know how the HTML and CSS models apply so we need to state something, and as HTML approaches to metadata do not operate on collections, we need to propose something there as well, lest we have no metadata on the collection.

We cannot possibly get everything right on the first attempt, but I don't think we can in good faith go to FPWD without at least an attempt at answering the question of how a Web Publication renders, including in a paged environment.

Reply:
(to be clear, my comment above was in reply to @lrosenthol)